models/apartments.rb

```diff
@@ -0,0 +1,18 @@
+class Apartment < ActiveRecord::Base
+  has_many :tenants
+  # attr_accessor :renters, :rent
+  # attr_reader :address, :sqft, :num_beds, :num_baths
+  # def initialize (address,rent,sqft,num_beds,num_baths,renters)
+  #   @address = address
+  #   @rent = rent
+  #   @sqft = sqft
+  #   @num_beds = num_beds
+  #   @num_baths = num_baths
+  #   @renters = renters
+  # end
+  # def add_tenant (tenant)
+  #   if @renters.length < @num_beds
+  #     @renters << tenant
+  #   end
+  # end
```

Comment:
if you uncomment this method `add_tenant`, you will have defined an instance method on your active record inherited Apartment class. Then you'll be able to do things like:

```rb
apt = Apartment.first
tenant = Tenant.first
apt.add_tenant(tenant)
```

Since this is an association, I would rely on active record methods to accomplish this.

A more practical application would be something like:

```rb
class Person < ActiveRecord::Base
  # in the db, person has a first name and last name columns
  def full_name
    self.first_name + " " + self.last_name
  end
end
```